### PR TITLE
feat(Tabs): migrate to Tailwind

### PR DESCRIPTION
**Pull request by mainframev**
Solves [FEPLT-1770](https://kiwicom.atlassian.net/browse/FEPLT-1770) 

# This Pull Request meets the following criteria:

### For tailwind migrations:

- [x] Visual regression test suite has been added and run before the migration.
- [x] `tailwind-migration-status.yaml` file has been updated with the migrated component(s).
- [x] Visual regression test suite has been run after the PR approval and no visual changes were identified.

[FEPLT-1770]: https://kiwicom.atlassian.net/browse/FEPLT-1770?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
 Storybook: https://orbit-mainframev-tw-migrate-tabs.surge.sh

### CODE REVIEW

**Comment by mainframev** [comment]
It's ready for review, except screenshots (I need to run it for CI, so it will generate linux screenshots), everything is ready for review, it's not a blocker for a review ;)

File: packages/orbit-design-tokens/output/docs-tokens.json [hunk 1]
```diff
@@ -15069,15 +15092,15 @@
     },
     "javascript": {
       "name": "tabBundleBasicBackground",
-      "value": "linear-gradient(80.91deg, #E13E3B 0%, #E87E09 100%)"
+      "value": "#FFFFFF"
```
Comment (oreqizer) [inline packages/orbit-design-tokens/output/docs-tokens.json:15095]: why are these changed?

**Review by oreqizer — CHANGES_REQUESTED** [review]
got some comments 🙏

File: packages/orbit-components/src/Tabs/components/Tab/index.tsx [hunk 2]
```diff
@@ -123,25 +25,68 @@ const Tab = ({
...
+      className={cx(
+        "flex flex-1 flex-row items-center justify-between",
+        "appearance-none flex-row border-0",
+        "min-w-fit",
+        "relative box-border",
+        "font-base font-medium",
+        "rounded-t-normal",
+        "duration-fast transition-[background] ease-in-out",
+        "px-md",
+        compact ? "text-normal py-[5px] leading-normal" : "text-large leading-large py-[9px]",
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/Tabs/components/Tab/index.tsx:39]: we may, it was added according to design spec, probably can ask designers to increase it to 6px on py and 8 px on py 🤔

File: packages/orbit-design-tokens/output/docs-tokens.json (hunk 1 above)
Comment (mainframev) [reply packages/orbit-design-tokens/output/docs-tokens.json:15095]: because technically background is white, and foreground (text) has the gradient, I adjusted it to reflect it's behaviour :)

File: packages/orbit-design-tokens/output/docs-tokens.json [hunk 3]
```diff
@@ -15057,6 +15057,29 @@
       "value": "rgb(0, 172, 238)"
     }
   },
+  "tabBundleBasicForeground": {
+    "type": "color",
+    "deprecated": false,
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-design-tokens/output/docs-tokens.json:15062]: nope, thanks!

File: packages/orbit-components/src/Tabs/components/Tab/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/src/Tabs/components/Tab/index.tsx:39]: yea later on, don't bother now!

File: packages/orbit-components/src/Tabs/Tabs.ct.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import { TabStory, TabsComponentStory, TabDisabledStory } from "./Tabs.ct-story";
+import { TYPE_OPTIONS } from "./components/Tab/consts";
+
+test.describe("visual Tab", () => {
+  Object.values(TYPE_OPTIONS).forEach(type => {
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/Tabs/Tabs.ct.tsx:8]: yes 👍🏻

File: packages/orbit-components/src/Tabs/components/Tab/index.tsx [hunk 5]
```diff
@@ -123,25 +25,68 @@ const Tab = ({
...
-      active={isSelected}
     >
-      <StyledTabText type={type}>{children}</StyledTabText>
-    </StyledTab>
+      <span
+        className={cx(
+          "text-ink-dark w-full",
+          type !== TYPE_OPTIONS.DEFAULT && [
+            "bg-clip-text [-webkit-background-clip:text] [-webkit-text-fill-color:transparent]",
+            type === TYPE_OPTIONS.BASIC && "bg-tab-bundle-basic-foreground",
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/Tabs/components/Tab/index.tsx:81]: I guess it's the matter of personal perference. I found it easier to read when it's in array, rather than having explicit longer condition for each of them, but I will apply as u suggesting, no strong opinion on this :)

File: packages/orbit-design-tokens/output/docs-tokens.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-design-tokens/output/docs-tokens.json:15062]: UPD: this is generated, so it's always true or false

File: packages/orbit-components/src/Tabs/components/TabList/index.tsx [hunk 6]
```diff
@@ -29,14 +16,42 @@ const TabList = ({
...
+  } as React.CSSProperties;
+
   return (
-    <StyledTabListWrapper
+    <div
+      className={cx(
+        fullWidth ? "w-full" : "w-auto",
+        cssVars["--tab-list-padding"] != null && "p-[var(--tab-list-padding)]",
+        cssVars["--tab-list-margin"] != null && "m-[var(--tab-list-margin)]",
+        cssVars["--tab-list-padding"] != null && [
```
Comment (oreqizer) [inline packages/orbit-components/src/Tabs/components/TabList/index.tsx:38]: this is actually false. if `--tab-list-padding` is not `undefined`, then `padding` is a `string`. in that case, the below vars (`...padding-top`, etc.) will be `undefined`

you need to check either if `padding` is an object, or preferably, if e.g. `padding-top` is not `undefined`

File: packages/orbit-components/src/Tabs/components/TabPanel/index.tsx [hunk 7]
```diff
@@ -1,26 +1,51 @@
...
+    "--tab-list-margin-left": typeof margin === "object" ? margin.left : undefined,
+  } as React.CSSProperties;
+
   return active || isActive ? (
-    <StyledPanel $margin={margin} $padding={padding} id={`panel-${index}`} data-test={dataTest}>
+    <div
+      className={cx(
+        cssVars["--tab-list-padding"] != null && "p-[var(--tab-list-padding)]",
+        cssVars["--tab-list-margin"] != null && "m-[var(--tab-list-margin)]",
+        cssVars["--tab-list-padding"] != null && [
```
Comment (oreqizer) [inline packages/orbit-components/src/Tabs/components/TabPanel/index.tsx:30]: same here, see above comment 🙏

File: packages/orbit-design-tokens/output/docs-tokens.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-design-tokens/output/docs-tokens.json:15062]: 👍

**Review by oreqizer — CHANGES_REQUESTED** [review]
one last thing 🙏

**Review by oreqizer — APPROVED** [review]
lgtm 👍 love em screenshots
